critiquebrainz/frontend/external/musicbrainz.py

```diff
@@ -20,6 +22,19 @@ def init(app_name, app_version, hostname=None):
         musicbrainzngs.set_hostname(hostname)
 
 
+def search_editon_group(query='', limit=None, offset=None):
```

Comment:
let's add a new `bookbrainz.py` for this, so that we can add the upcoming search methods to it.